recipes/lora_finetune_distributed.py

```diff
@@ -279,16 +281,13 @@ def _setup_model(
         """
 
         if self._is_rank_zero:
-            log.info("FSDP is enabled. Instantiating Model on CPU for Rank 0 ...")
+            log.info("FSDP is enabled. Model init and checkpoint loading on Rank 0 ...")
             init_start = time.perf_counter()
 
-            with utils.set_default_dtype(self._dtype):
-                model = config.instantiate(cfg_model)
-
-            log.info(
-                f"Model instantiation took {time.perf_counter() - init_start:.2f} secs"
-            )
+        with utils.set_default_dtype(self._dtype), torch.device("meta"):
```

Comment:
the docstring used to be `Instantiating Model on CPU` (left) and I removed the mention of CPU. I did not mention meta device because it measures meta init + checkpoing loading now. Happy to improve if you are referring to this docstring

Reply:
oh. Just got you point. Updated docstring for `_setup_model`